klayout/drc/rule_decks/dummy_exclude.drc

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+#================================================
+#--------------DUMMY EXCLUDE LAYERS--------------
+#================================================
+
+# Rule DE.1 is not a DRC check
+## Please refer to https://gf180mcu-pdk.readthedocs.io/en/latest/physical_verification/design_manual/drm_10_08.html#design-rules-for-dummy-exclude-layers-ndmy-and-pmndmy
+
+# Rule DE.2: Minimum NDMY or PMNDMY size (x or y dimension in um) is 0.8µm.
+logger.info('Executing rule DE.2')
+de2_l1 = ndmy.or(pmndmy).width(0.8.um, euclidian)
+de2_l1.output('DE.2', 'DE.2 : Minimum NDMY or PMNDMY size (x or y dimension in um) : 0.8µm')
+de2_l1.forget
```

Comment:
nit: should we set those to nil (as suggested in https://www.klayout.de/doc/about/drc_ref_layer.html#h2-1260) to make sure they are not being used anymore.

Reply:
https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pv/issues/53